### PR TITLE
Cards block for VIP Area

### LINKED ISSUE #14: Cards block for VIP Area
Implement cards block for VIP Area.
You can see it in this page of the original website: https://www.realmadrid.com/area-vip/partido-vip towards the end of the page

Screenshot 2023-04-26 at 18 23 10

Screenshot 2023-04-26 at 18 22 39

### PR DESCRIPTION
**Pull request by iuliag**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #14 

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/drafts/grumaz/partido-vip
- After: https://14-cards-block--realmadrid--hlxsites.hlx.page/drafts/grumaz/partido-vip

### CODE REVIEW

File: blocks/vip-sections/vip-sections.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    if (pdfAnchor) {
+      pdfAnchor.setAttribute('download', 'download');
+    }
+
+    const anchor = section.querySelector('p > a:first-of-type');
+    if (!anchor || anchor.href.endsWith(excludedSectionPath)) {
+      return; // exclude current section, exclude sections without path
+    }
+
+    const breakpoints = [
```
Comment (chicharr) [inline blocks/vip-sections/vip-sections.js:42]: It looks like the images don't have enough resolution on mobile.
In this case the image must be larger when the screensize is smaller. I checked the original website and they are using. 
960px for mobile
640px for desktop

File: blocks/vip-sections/vip-sections.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ * @param {string} path The path to the fragment
+ * @returns {HTMLElement} The root element of the fragment
+ */
+async function loadFragment(path) {
+  if (path && path.startsWith('/')) {
+    const resp = await fetch(`${path}.plain.html`);
+    if (resp.ok) {
+      const main = document.createElement('main');
+      main.innerHTML = await resp.text();
+      // decorateMain(main);
```
Comment (chicharr) [inline blocks/vip-sections/vip-sections.js:18]: remove comments if they are not needed

**Review by chicharr — APPROVED** [review]
@iuliag it looks good to me. 
just 2 small comments

File: blocks/vip-sections/vip-sections.js (hunk 1 above)
Comment (iuliag) [reply blocks/vip-sections/vip-sections.js:40]: Increased resolution for <990px.
I may be looking in a different place on the original website, but I see 3 widths for the images: 960px for screen up to 990px, 480px - from 990px to 1200px and 600px from 1200px upwards. So I adjusted for these.

**Pull request merged**